sshd/server.go

```diff
@@ -93,6 +93,7 @@ func (s *SSHServer) Run(addr string) error {
 	var err error
 	s.listener, err = net.Listen("tcp", addr)
 	if err != nil {
+		s.l.WithField("err", err).Warn("Error starting SSH server listener, shutting down")
```

Comment:
Caller should be the one to log

Reply:
Moved the log line, and verified that I saw the error log when misconfiguring the ssh listener.